L1Trigger/L1TCaloLayer1/plugins/L1TCaloLayer1.cc

```diff
@@ -352,10 +352,21 @@ void L1TCaloLayer1::beginRun(const edm::Run& iRun, const edm::EventSetup& iSetup
 
 // ------------ method fills 'descriptions' with the allowed parameters for the module  ------------
 void L1TCaloLayer1::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
-  //The following says we do not know what parameters are allowed so do no validation
-  // Please change this to state exactly what you do use, even if it is no parameters
+  //Description set to reflect default present in simCaloStage2Layer1Digis_cfi.py
+  //Currently redundant, but could be adjusted to provide defaults in case additional LUT
+  //checks are added and before other configurations adjust to match.
   edm::ParameterSetDescription desc;
-  desc.setUnknown();
+  desc.add<edm::InputTag>("ecalToken", edm::InputTag("simEcalTriggerPrimitiveDigis"));
+  desc.add<edm::InputTag>("hcalToken", edm::InputTag("simHcalTriggerPrimitiveDigis"));
+  desc.add<bool>("useLSB", true);
+  desc.add<bool>("useCalib", true);
+  desc.add<bool>("useECALLUT", true);
+  desc.add<bool>("useHCALLUT", true);
+  desc.add<bool>("useHFLUT", true);
+  desc.add<bool>("verbose", false);
```

Comment:
This should be made preferably untracked.
However, in order to speed up the updates in #41221 I will merge it as such. But please @vshang when you will update this method in #41221 please make it untracked, i.e.
```suggestion
  desc.addUntracked<bool>("verbose", false);
```
and modify https://github.com/cms-sw/cmssw/blob/60499d5eac1799b4855d61b7b948a2930278a188/L1Trigger/L1TCaloLayer1/python/simCaloStage2Layer1Digis_cfi.py#L17 accordingly